src/main/resources/org/jenkinsci/plugins/workflow/cps/replay/Messages.properties

```diff
@@ -1,2 +1,4 @@
 Replay.permission.description=Ability to perform a new Pipeline build with an edited script.
 ReplayCommand.shortDescription=Replay a Pipeline build with edited script taken from standard input
+Replay=Replay
```

Comment:
It looks like these keys being capitalized is causing the build to fail. Maybe rename them to `ReplayAction.displayName` and `ReplayCause.shortDescription`?